language/move-binary-format/src/file_format.rs

```diff
@@ -92,6 +92,30 @@ macro_rules! define_index {
                 self.0 as usize
             }
         }
+
+        impl Serialize for $name {
```

Comment:
I think we should have a feature flag for this to avoid the serialization to be used anywhere else

Reply:
I try to add a feature flag but found that if the Index type need a feature flag, most struct also needs the flag. Is it worth to do?